[2017/12/20 17:27:12] :: [pool-1-thread-9] :: Base path for the test = C:\New Selenium\Salesforce_QA_Automation\ADT_Resources\<br><br>[2017/12/20 17:27:12] :: [pool-1-thread-9] :: Current Thread Name for the test is "pool-1-thread-9"<br><br>[2017/12/20 17:27:22] :: [pool-1-thread-9] :: The user has launched 'https://qainc--qa.cs30.my.salesforce.com<br><br>[2017/12/20 17:27:23] :: [pool-1-thread-9] :: The user has creating RetCreditNONorder<br><br>[2017/12/20 17:27:37] :: [pool-1-thread-9] :: arindom.das@move.com.qa has successfully loggedin<br><br>[2017/12/20 17:27:38] :: [pool-1-thread-9] :: Successfully navigated to Home page<br><br>[2017/12/20 17:27:39] :: [pool-1-thread-9] :: File path is = C:\New Selenium\Salesforce_QA_Automation\ADT_Resources\ .<br><br>[2017/12/20 17:29:48] :: [pool-1-thread-9] :: Successfully Account ID is visible= 001j000000bXMiR .<br><br>[2017/12/20 17:30:13] :: [pool-1-thread-9] :: Successfully clicked on GlobalSearchBtn .<br><br>[2017/12/20 17:30:48] :: [pool-1-thread-9] :: Successfully clicked on PeopleName .<br><br>[2017/12/20 17:31:07] :: [pool-1-thread-9] :: Successfully clicked on FollowDown .<br><br>[2017/12/20 17:31:20] :: [pool-1-thread-9] :: Successfully clicked on UserDetail .<br><br>[2017/12/20 17:31:40] :: [pool-1-thread-9] :: Successfully clicked on LoginUsers .<br><br>[2017/12/20 17:32:27] :: [pool-1-thread-9] :: Tab is not available,AccountDetails<br><br>[2017/12/20 17:32:51] :: [pool-1-thread-9] :: Successfully clicked on NewCase .<br><br>[2017/12/20 17:33:16] :: [pool-1-thread-9] :: Successfully clicked on Continue .<br><br>[2017/12/20 17:33:27] :: [pool-1-thread-9] :: Successfully Enter on SubTitle button<br><br>[2017/12/20 17:33:31] :: [pool-1-thread-9] :: Successfully Enter on Description button<br><br>[2017/12/20 17:33:43] :: [pool-1-thread-9] :: Successfully clicked on AssignCheckBox .<br><br>[2017/12/20 17:34:00] :: [pool-1-thread-9] :: Successfully clicked on CSave .<br><br>[2017/12/20 17:34:30] :: [pool-1-thread-9] :: Successfully clicked on ChangeCaseOwnerName .<br><br>[2017/12/20 17:34:39] :: [pool-1-thread-9] :: Successfully case owner name is enter-Emily Morrison .<br><br>[2017/12/20 17:34:55] :: [pool-1-thread-9] :: Successfully clicked on CSave .<br><br>[2017/12/20 17:35:31] :: [pool-1-thread-9] :: Successfully clicked on AddAssets .<br><br>[2017/12/20 17:35:35] :: [pool-1-thread-9] :: product and CancellationReasons are not selected<br><br>[2017/12/20 17:35:42] :: [pool-1-thread-9] :: Successfully Add Selected Assets To The Case-SubmitButton .<br><br>[2017/12/20 17:36:04] :: [pool-1-thread-9] :: Successfully clicked on LogoutProfile .<br><br>[2017/12/20 17:36:19] :: [pool-1-thread-9] :: Successfully clicked on Logout .<br><br>[2017/12/20 17:36:48] :: [pool-1-thread-9] :: Successfully clicked on GlobalSearchBtn .<br><br>[2017/12/20 17:37:21] :: [pool-1-thread-9] :: Successfully clicked on FollowDown .<br><br>[2017/12/20 17:37:34] :: [pool-1-thread-9] :: Successfully clicked on UserDetail .<br><br>[2017/12/20 17:37:57] :: [pool-1-thread-9] :: Successfully clicked on LoginUsers .<br><br>[2017/12/20 17:38:23] :: [pool-1-thread-9] :: Successfully clicked on GlobalSearchBtn .<br><br>[2017/12/20 17:38:46] :: [pool-1-thread-9] :: Successfully case number searched-02069859 .<br><br>[2017/12/20 17:39:20] :: [pool-1-thread-9] :: Successfully click on Create Retension Quote-CreateRetentionQuote .<br><br>[2017/12/20 17:40:10] :: [pool-1-thread-9] :: Exception occurred. null value in entry: id=null<br><br>[2017/12/20 17:40:13] :: [pool-1-thread-9] :: Closing the Browser Instance<br><br>[2017/12/20 17:40:13] :: [pool-1-thread-9] ::  File Name: Assert.java Method name: fail(86)<br> File Name: Assert.java Method name: assertTrue(41)<br> File Name: Assert.java Method name: assertTrue(52)<br> File Name: CancelOrder.java Method name: addSelectItemsOrder(569)<br> File Name: C:/New Selenium/Salesforce_QA_Automation/ADT_Resources/SalesForce/FeatureFiles/AMD1/Order_Credit_DomainName_XL.feature Method name: And I am selecting Items for Retention "Credit" and "10" and "NoCancelType" and "NoCancellationReason"(13)<br><br>Complete Error Message: null<br><br><br>